Fixed phone number related problem (#301)
AY2324S2-CS2103-F15-1/tp: docs/UserGuide.md

  * All input values are case-sensitive unless otherwise specified.

* Duplicate contacts:

> * In FINDvisor, contacts are **uniquely identified** using their mobile phone number. Hence, two contacts are considered duplicates if they share the same mobile phone number. 

Reviewer: I think you are missing an indentation here, also remove the trailing space.

>   * Contacts are considered duplicates if they share the same mobile phone number and is used as unique identifier of a Person as no two persons would share the same mobile phone number.

Reviewer 2: ```suggestion
* In FINDvisor, contacts are **uniquely identified** using their mobile phone number. Hence, two contacts are considered duplicates if they share the same mobile phone number. 
```
@KohGuanZeh @YuZizhen Just a nit, but does this sound better?

Reviewer 3: Looks good to me. I think we should adjust it to this.

  * All input values are case-sensitive unless otherwise specified.

* Duplicate contacts:

>   * Contacts are considered duplicates if they share the same mobile phone number.
>   * We consider mobile phone number as an unique identifier of a person in Singapore because in most of the cases, two people will not share the same mobile number (similar to how PayLah and PayNow works)

Reviewer 3: ```suggestion
  * Contacts are considered duplicates if they share the same mobile phone number and is used as unique identifier of a Person as no two persons would share the same mobile phone number.
```
Would this be better?
